build/package.yml

```diff
@@ -3,10 +3,18 @@ steps:
   # Package web
   
   - task: DotNetCoreCLI@2
-    displayName: 'dotnet publish web'
+    displayName: 'dotnet publish Stu3 web'
     inputs:
       command: publish
-      projects: '**/Microsoft.Health.Fhir.Web.csproj'
+      projects: '**/Microsoft.Health.Fhir.Stu3.Web.csproj'
+      arguments: '--runtime win10-x64 --output $(build.artifactStagingDirectory)/web --configuration $(buildConfiguration) --version-suffix $(build.buildnumber)'
```

Comment:
nit: since the only difference is the displayName and projects, maybe consider moving this task into a template so it can be reused or at least make the arguments into a variable and reuse here so that if we need to update, it will be in one place?